src/recon_buildblock/BackProjectorByBin.cxx

```diff
@@ -200,11 +200,12 @@ BackProjectorByBin::back_project(const ProjData& proj_data, int subset_num, int
                                          subset_num, num_subsets);
 
 #ifdef STIR_OPENMP
-#pragma omp parallel shared(proj_data, symmetries_sptr)
-#endif
-  {
-#ifdef STIR_OPENMP
-#pragma omp for schedule(dynamic)
+  #if _OPENMP <201107
```

Comment:
I got this OpenMP version from [KOSMAPOSLReconstruction](https://github.com/UCL/STIR/blob/4e016d62805754ad6fb4e920295f6e76dde3f6ea/src/iterative/KOSMAPOSL/KOSMAPOSLReconstruction.cxx#L716-L722) where a `collapse(3)`is used.